res/templates/login.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Login</title>
+</head>
+<body>
+<h1>Login</h1>
+<form action="/login" method="POST">
+    <input type="text" name="username"/>
+    <input type="password" name="password"/>
+    <button type="submit">
+        Login
+    </button>
+</form>
+</body>
+</html>
```

Comment:
It's recommended to name them .mustache; the templates I have locally are named .mustache ...?